sharding-distribution/sharding-scaling-distribution/src/main/resources/bin/start.sh

```diff
@@ -20,8 +20,10 @@ cd `dirname $0`
 cd ..
 DEPLOY_DIR=`pwd`
 
+JAVA_OPTS=" -Dport=8080 "
+
 CLASS_PATH=.:${DEPLOY_DIR}/conf:${DEPLOY_DIR}/lib/*
 
 MAIN_CLASS=org.apache.shardingsphere.shardingscaling.Bootstrap
 
-java -classpath ${CLASS_PATH} ${MAIN_CLASS}
\ No newline at end of file
+java ${JAVA_OPTS} -classpath ${CLASS_PATH} ${MAIN_CLASS}
```

Comment:
Moving port to configFile maybe better. You can refer `Bootstrap in sharding-proxy`.